app/celer-sim/RunnerInputIO.json.cc

```diff
@@ -178,6 +180,8 @@ void to_json(nlohmann::json& j, RunnerInput const& v)
     LDIO_SAVE_OPTION(slot_diagnostic_prefix);
     LDIO_SAVE(write_track_counts);
     LDIO_SAVE(write_step_times);
+    LDIO_SAVE_OPTION(transporter_result);
+    LDIO_SAVE_OPTION(print_progress);
```

Comment:
Just use `LDIO_SAVE` since their value is meaningful even when not explicitly set.